presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+* Improve result comparison for orderable array columns by applying :func:`array_sort` before :func:`checksum`.
+* Improve result comparison for ``Select`` queries by rewriting ``Select`` queries as ``CreateTableAsSelect`` and using checksum queries to verify the results. Therefore, eliminate the row count limit for ``Select`` queries.
+* Improve determinism check by reusing the initial control query results, and thus reducing the maximum numbers of control query rerun from 3 to 2. Also, eliminate test query reruns.
+* Improve Verifier interfaces to allow easy extension by community users.
+* Add support for retrying transient Presto query failures, using configuration properties ``presto.max-attempts``, ``presto.min-backoff-delay``, ``presto.max-backoff-delay``, ``presto.backoff-scale-factor``. Similar configurations prefixed with ``cluster-connection`` instead of ``presto`` are introduced for retrying transient network failures when talking to Presto server. Intermediate failures are recorded and emitted in the output.
+* Add support for automatically resolving failures, including exceeding global memory limit and exceeding time limit.
+* Add configuration properties ``metadata-timeout`` and ``checksum-timeout`` to set the timeouts for metadata queries (i.e. describe queries that reads table schema) and checksum queries.
+* Add configuration property ``source-query.table-name`` to allow the table name of the MySQL table from which verifier queries are loaded to be configured.
+* Add configuration property ``human-readable.log-file`` to allow human-readable events to be logged into the specified file instead of to ``stdout``.
+* Rename configuration property ``query-database`` to ``source-query.database``.
+* Rename configuration property ``suites`` to ``source-query.suites``.
+* Rename configuration property ``max_queries`` to ``source-query.max-queries-per-suite``.
+* Rename configuration property ``event-client`` to ``event-clients``.
+* Rename configuration property ``event-log-file`` to ``json.log-file``.
+* Rename configuration property ``run-id`` to `test-id`.
+* Rename configuration property ``thread-count`` to ``max-concurrency``.
+* Rename configuration properties ``control-gateway`` to ``control.jdbc_url``, and ``test-gateway`` to ``test.jdbc_url``.
+* Rename configuration properties ``shadow-writes.control-table-prefix`` to ``control.table-prefix``, and ``shadow-writes.test-table-prefix`` to ``test.table-prefix``.
+* Remove configuration properties ``control.query-types``, ``test.query-types``, ``source``, ``max-row-count``, ``always-report``, ``skip-correctness-regex ``, ``check-correctness ``, ``skip-cpu-check-regex ``, ``check-cpu``, ``explain-only``, ``verbose-results-comparison``, ``quiet``, ``control-teardown-retries``, ``test-teardown-retries``, ``shadow-writes``.
```

Comment:
@nezihyigitbasi I wonder if there're too much details in the verifier section, but I wonder if/how I should shrink it down.